compute_tools/src/http/routes/configure.rs

```diff
@@ -0,0 +1,75 @@
+use std::sync::Arc;
+
+use axum::{extract::State, response::Response};
+use compute_api::{
+    requests::ConfigurationRequest,
+    responses::{ComputeStatus, ComputeStatusResponse},
+};
+use http::StatusCode;
+use tokio::task;
+use tracing::info;
+
+use crate::{
+    compute::{ComputeNode, ParsedSpec},
+    http::{extract::Json, JsonResponse},
+};
+
+pub(in crate::http) async fn configure(
+    State(compute): State<Arc<ComputeNode>>,
+    request: Json<ConfigurationRequest>,
+) -> Response {
```

Comment:
There were various comments in the previous version of this function, can you please transfer them into new code as well?

Overall, all new handlers look heavily under-commented, so I guess some existing comments can be missing in other places